app/src/main/java/com/sschuraytz/charcoaldrawing/SmudgeTool.java

```diff
@@ -41,29 +40,23 @@ public SmudgeTool() {
      */
     @Override
     protected void onDown(Canvas bitmapCanvas, Bitmap inputBitmap, float x1, float y1) {
-        smudgePaint.setAlpha(this.getOpacity());
+        paint.setAlpha((int) (this.getOpacity() / 3));
```

Comment:
this results in the smudge effect ending pretty quickly. How about dividing by 2 instead of 3?

Reply:
let's split the difference and go to 2.5. I thought 3 was too long